docs/resources/application_agent_credential.md

```diff
@@ -54,6 +51,7 @@ Optional:
 - `default` (String)
 - `delete` (String)
 - `read` (String)
+- `update` (String)
```

Comment:
This is wrong. There is no update for Credential.

Reply:
updated